confd/pkg/backends/calico/routes_test.go

```diff
@@ -12,6 +12,9 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/cache"
+
+	apiv3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
```

Comment:
`make check-fmt` in confd/ seems to be failing in this file. Try running `make fix` and commiting if there are changes...